translate/expression/py_expression.py

```diff
@@ -26,10 +26,10 @@ def call(node):
 """
 takes ast.name node from python ast and converts to string 
 represenation for the generic ast
+FIXME: this should prob be in helpers since it is also used by assign
 """
 def name(node):
     if node.id == "print":
         return "logStatement"
     else:
-        return "customStatement"
-      
\ No newline at end of file
+        return node.id      
```

Comment:
Shouldn't this still be a type?

If we want to return the ID it should probably be in an additional field that exists on `customStatement` objects.

Reply:
added a FIXME- i think this plays into the broader types issue that we are going to face